components/infobox/commons/infobox_league.lua

```diff
@@ -477,4 +479,15 @@ function League:_fetchAbbreviation()
 	end
 end
 
+-- Given a table, resolves all redirects in any value of the table
+function League:_resolveAllRedirectsInTable(table)
+	if table == nil then return nil end
+
+	for key, value in pairs(table) do
+		table[key] = mw.ext.TeamLiquidIntegration.resolve_redirect(value)
+	end
+
+	return table
```

Comment:
```suggestion
	return Table.mapValues(table, mw.ext.TeamLiquidIntegration.resolve_redirect)
```

Reply:
Even better, don't have this function at all and put it on line 254 instead

Reply:
TIL about `Module:Table`. Will change.